### DISCUSSION TITLE
how to start artisan.py fullscreen from cli

### DISCUSSION THREAD

**Discussion by ju-emb** [artisan-roaster-scope/artisan#1681] [Q&A] [answered]
Hello,

how to start artisan.py in fullscreen from the command line in linux

**Comment by MAKOMO** [comment, marked as answer]
Independent how you start Artisan, from source or from a build, it should first read its app settings from the system. Those settings are written on quitting the app. Thus if you bring Artisan in full-screen mode, quit the app and restart, it starts up in full screen mode.

**Reply by ju-emb** [reply]
Thanks Makomo,

that did the trick.
